dependencies/shotgun_api3/lib/simplejson/decoder.py

```diff
@@ -177,10 +180,11 @@ def py_scanstring(s, end, encoding=None, strict=True,
 WHITESPACE = re.compile(r'[ \t\n\r]*', FLAGS)
 WHITESPACE_STR = ' \t\n\r'
 
-def JSONObject((s, end), encoding, strict, scan_once, object_hook,
+def JSONObject(xxx_todo_changeme, encoding, strict, scan_once, object_hook,
```

Comment:
Whats up with the variable name here?

Reply:
The modernizer is set to moved the tuple (s,end) to inside the function. The previous function definition is no longer valid in python 3

Reply:
I will change name to something else